tests/src/test/java/alluxio/client/cli/fs/command/CopyFromLocalCommandIntegrationTest.java

```diff
@@ -207,9 +207,8 @@ public void copyFromLocal() throws IOException, AlluxioException {
         BufferUtils.getIncreasingByteArray(10, 20));
 
     mFsShell.run("copyFromLocal", testFile.getParent(), "/testDir");
-    Assert.assertEquals(
-        getCommandOutput(new String[]{"copyFromLocal", testFile.getParent(), "/testDir"}),
-        mOutput.toString());
+    Assert.assertTrue(mOutput.toString().contains(
```

Comment:
for asserting string contains, it's better to use `assertThat(mOutput.toString(), containsString(X))`

then if it fails, it will report the actual string and the expected string, making it easier to debug

Reply:
good to know